AbilityDeck.jsx

```diff
@@ -12,6 +12,22 @@ import { MONSTER_STATS } from './monster_stats';
 
 import * as css from './style/Card.scss';
 
+function StatIcon(props) {
+  const parse = /^%?([^%]+)%?$/;
+  const match = parse.exec(props.name);
+  const imgSource = `images/${match[1]}.svg`;
```

Comment:
I think we need a special case here for the "flying" attribute, since the image is named fly.svg.  Alternatively, I would be supportive of just renaming it to flying.svg, since it's just referenced in one other place (macros.js).

Reply:
fwiw - I had considered bumping StatIcon into its own file/component and then adding a rather large LUT which would would allow me to remap stat/abilities to the proper icon filename, and then also to add some rich alt text which could do things like remind the user of the specific ability rule along with a page number reference when deep rules lawyer-ing was needed.

I may still end up doing this, but for now, this seems like it might be out of scope for this CL.  I think I'll end up adding a special case for now; both because I might do the LUT anyway someday, and because I would rather not diverge too much from the upstream.  macros.js right now are leftovers from the orignial, so I'd like to avoid introducing diffs whenever I can.